### PR TITLE
adds resume intent

### PR DESCRIPTION
**Pull request by tsop14**
If the user was in the middle of a podcast episode when they last said "Alexa
stop", when they say "Alexa resume" it will continue playing the episode.

### CODE REVIEW

**Comment by mchv** [comment]
Is there a way to add a test to check its works as intended?

**Comment by tsop14** [comment]
@mchv deploying to CODE and testing using Alexa app.

**Comment by mchv** [comment]
Code looks logical to me 👍

File: src/index.js [hunk 1]
```diff
@@ -60,9 +61,7 @@ var commonHandlers = {
     this.emit(':responseReady')
```
Comment (mchv) [inline src/index.js]: Just for knowing, what does `this.emit(':responseReady')` exactly do behind the scene?

Comment (tsop14) [inline src/index.js]: "this.emit(':responseReady'); // Called after the response is built but before it is returned to the Alexa service. Calls :saveState."

https://developer.amazon.com/blogs/post/Tx213D2XQIYH864/announcing-the-alexa-skills-kit-for-node-js

**Pull request merged**